data/mods/blazemod/blaze_test.json

```diff
@@ -235,7 +235,7 @@
         "price" : 1500000,
 		"watertight": true,
 		"seals": true,
-		"contains": 0
+		"contains": 1
```

Comment:
Comments on the forum suggest the original author is maintaining a version separately outside the main tree? I'm not especially up to date with the forums so can anyone shed any more light on this?